### Rauf Salamzade - Grad Student @ University of Wisconsin - Madison / Kalan Lab

Hi there, my name is Rauf, I am a doctorate student in microbiology at the University of Wisconsin - Madison, working within the Kalan Lab. Prior to that, I worked in the Earl Bacterial Genomics group at the Broad Institute, the functional data science team at Bayer CropScience Biologics, and briefly within the Arkin lab at LBNL/UC Berkeley. Some software I have developed include:

:snake: = *on Bioconda*; :whale2: = *on Docker Hub*; :pie::pie: = *on PyPi*

#### Active development:

* [*zol (& fai)*](https://github.com/Kalan-Lab/zol): :snake: :whale2: suite for evolutionary and functional analysis of gene clusters (co-located sets of genes).
* [skDER (& CiDDER)](https://github.com/raufs/skDER): :snake: programs for genomic dereplication using ANI and pan-genome saturation approximation.
* [*lsa*BGC-Pan](https://github.com/Kalan-Lab/lsaBGC-Pan): :snake: suite/workflow for pan-BGC-ome analyses.
* [codoff](https://github.com/Kalan-Lab/codoff): :snake::pie::pie:   program to statistically assess differences in codon usage between a focal region and the background genome.
  
#### Supported but no longer in active development:

* [*lsa*BGC](https://github.com/Kalan-Lab/lsaBGC): :whale2: suite for genomic and metagenomic mining of micro-diversity in BGCs and general evolutionary analysis.
* [ConSequences](https://github.com/broadinstitute/ConSequences): programs to find long (>10 kb) contiguous and conserved (> 99% identity) segments across genomes. 
* [seQouia](https://github.com/broadinstitute/sequoia): framework for diverse omics workflows, primarily focused on QC, processing and analysis of bacterial (meta-)genomic sequencing datasets. 
* [AncestralGeneRator](https://github.com/broadinstitute/ancestralgenerator): programs to simplify ancestral state reconstruction of ortholog group copy counts across phylogeies. 
* [PerMutation](https://github.com/broadinstitute/permutation): Programs for statistical determination of essential and conditionally essential genes from *E. faecalis* TnSeq datasets.
